docs/analytics/aws.md

```diff
@@ -0,0 +1,30 @@
+# AWS Analytics: Info & Changelog
+
+_For a deeper technical writeup of analytics processes intended for developers, please see [aws-technical](aws-technical.md)._
+
+Trezor Suite can be set to collect real-world data to improve the performance of both web and desktop apps. This anonymous data is only shared by users who have usage data tracking enabled.
+
+Data tracking can be toggled on or off at any time through the Trezor Suite settings menu.
+
+By default, analytics will be enabled immediately following the onboarding process, and the application will start to track events and errors. Users are given the option to disable data collection before completing the onboarding.
```

Comment:
It is enabled immediately after you confirm
![Screenshot 2022-03-15 at 17 03 08](https://user-images.githubusercontent.com/33235762/158420259-848c4d77-e1cd-4e58-a0dd-6b402aeeead4.png)

Reply:
954586dd6a